Header header
std_msgs/Int32 centroid
std_msgs/Bool existe
